lib/ui/status_bar_widget.dart

```diff
@@ -60,7 +60,7 @@ class _StatusBarWidgetState extends State<StatusBarWidget> {
               AnimatedOpacity(
                 opacity: _showStatus ? 0 : 1,
                 duration: const Duration(milliseconds: 1000),
-                child: const StatusBarBrandingWidget(),
+                child: const StatusBarBrandingAndSearchWidget(),
```

Comment:
Having `and` in the code is always a bad sign 😬 
If we can't split these further, let's call this a `TopBarWidget` or something?